inc/taxonomies-functions.php

```diff
@@ -1977,17 +1977,44 @@ function taxopress_unregister_taxonomy($taxonomy)
 
     global $wp_taxonomies;
 
-    $taxonomy_object->remove_rewrite_rules();
-    $taxonomy_object->remove_hooks();
+    // Instead of removing totally, keep minimal structure but disable functionality
+   if (isset($wp_taxonomies[$taxonomy])) {
+        // Store original in case we need to restore
+        $original_tax = $wp_taxonomies[$taxonomy];
+        
+        $wp_taxonomies[$taxonomy] = new stdClass();
```

Comment:
@Deji98 Is there any reason you're creating new object instead of updating existing taxonomy data?